### PR TITLE
relocate io.netty and com.google to resolve lib conflict

### PR DESCRIPTION
**Pull request by yangb8**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -80,4 +80,7 @@ shadowJar {
     version = version
     zip64 = true
     mergeServiceFiles()
+
+    relocate 'io.netty', 'io.netty.pravega'
```
Comment (EronWright) [inline build.gradle]: Can we use a package name like `io.pravega.shaded.io.netty`

**Comment by EronWright** [comment]
Would you mind elaborating on why this was necessary?  I agree that some variation is needed; I was hoping that Pravega client itself would relocate its dependencies.  Is this basically a workaround for that?

**Comment by yangb8** [comment]
@EronWright 
I don't see pravega client relocates its dependencies. Hadoop as well as Spark are using some packages in lower versions provided by com.google and io.netty. Hadoop would use its own libraries unless some environment flag like HADOOP_USER_CLASSPATH_FIRST is set. Even worse, sometimes, called functions only exist in some specific version, so to pick up the package with higher version can't resolve the conflict.

**Review by EronWright — APPROVED** [review]

**Comment by yangb8** [comment]
@EronWright @fpj seems like I don't have write permission to merge the code. Could you please grant the permission to me?

**Pull request merged**